src/main/java/edu/usf/sas/pal/muser/ProcessorMain.java

```diff
@@ -35,22 +35,31 @@ public static void main(String[] args) {
 
         try {
             CommandLine cmd = parser.parse(options, args);
+
             if (cmd.hasOption(ProgramOptions.KEY_FILE)) {
                 programOptions.setFileKeyPath(cmd.getOptionValue(ProgramOptions.KEY_FILE));
             } else {
                 System.err.println("Firebase admin key is not provided. \n" +
                         "Provide an admin key using -keyFile path/to/file.json");
                 return;
             }
+
             if (cmd.hasOption(ProgramOptions.USER_ID)) {
                 programOptions.setUserId(cmd.getOptionValue(ProgramOptions.USER_ID));
             }
+
+            if (cmd.hasOption(ProgramOptions.START_DATE)) {
```

Comment:
Looking at the later code, it seems like start date and end date must both be provided together in order for them to be used. Please add this to the documentation and check here if both variables have been provided. If only one has been provided you should stop and output an error message.